mreg_cli/utilities/api.py

```diff
@@ -540,7 +540,7 @@ def get_list_generic(
     if expect_one_result:
         if len(ret) == 0:
             return {}
-        if len(ret) != 1:
+        if len(ret) > 1 and not all(ret[0] == x for x in ret):
```

Comment:
Suggest changing `not all` to `any` and thus the `==` to `!=`.

Reply:
Yes, that short circuits faster, you are right.